db/attachment_create.sql

```diff
@@ -0,0 +1,19 @@
+-- Table: public.attachment
+
+-- DROP TABLE public.attachment;
```

Comment:
After consideration, since these CREATE TABLE statements are executed externally via a CLI, we determined separation of the revert scripts are not necessary for our set up.